### PR TITLE
DX-1500: Payment Menu in production (develop)

### PR DESCRIPTION
**Pull request by asbjornu**
#840 but for `develop`.

### CODE REVIEW

File: .github/workflows/deploy_to_azure.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Deploy to stage and prod
```
Comment (asbjornu) [inline .github/workflows/deploy_to_azure.yml:1]: Strange that this change is coming from `master` when merging to `develop`, don't you agree, @NullableInt?

**Review by arebra — APPROVED** [review]
Looks good to me!

File: .github/workflows/deploy_to_azure.yml (hunk 1 above)
Comment (NullableInt) [reply .github/workflows/deploy_to_azure.yml:1]: I don't quite get what you are asking for here.

Comment (asbjornu) [reply .github/workflows/deploy_to_azure.yml:1]: Mystery solved, #839 was never merged to `develop`.

**Review by NullableInt — APPROVED** [review]

**Pull request merged**